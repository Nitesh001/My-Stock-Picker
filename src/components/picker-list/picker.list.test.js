import React from 'react';
import renderer from 'react-test-renderer';
import PickerList from './picker.list';

test('List shows items with values', () => {
  const component = renderer.create(
    <PickerList list={[{key:'k1',value:'v1'},{key:'k2',value:'v2'}]} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('List shows onclick function', () => {
  const component = renderer.create(
    <PickerList list={[{key:'k1',value:'v1'},{key:'k2',value:'v2'}]} onClick={(k)=>{}} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});