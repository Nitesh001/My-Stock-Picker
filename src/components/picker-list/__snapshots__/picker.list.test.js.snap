// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List shows items with values 1`] = `
<div
  className="list-group"
  onKeyDown={[Function]}
>
  <button
    className="list-group-item list-group-item-action"
    data-rr-ui-event-key={null}
    onClick={[Function]}
  >
    v1
  </button>
  <button
    className="list-group-item list-group-item-action"
    data-rr-ui-event-key={null}
    onClick={[Function]}
  >
    v2
  </button>
</div>
`;

exports[`List shows onclick function 1`] = `
<div
  className="list-group"
  onKeyDown={[Function]}
>
  <button
    className="list-group-item list-group-item-action"
    data-rr-ui-event-key={null}
    onClick={[Function]}
  >
    v1
  </button>
  <button
    className="list-group-item list-group-item-action"
    data-rr-ui-event-key={null}
    onClick={[Function]}
  >
    v2
  </button>
</div>
`;
